expressEngine/user/service/user.js

```diff
@@ -7,13 +7,16 @@ import { User } from '../../model';
 // import messages
 import { error, success } from '../../cms/user';
 
+// import utils
+import UserHelper from '../utils';
```

Comment:
@yogeshsinghh please use it as camelCase i.e. userHelper

Reply:
sir, I created a new method findAll for just because if we use find then there might be a lot of conflicts for another PR that uses find methods because findAll methods which take the arguments as an object

Reply:
done sir